lib/pdfjs_viewer-rails.rb

```diff
@@ -7,7 +7,7 @@ class Engine < ::Rails::Engine
       isolate_namespace PdfjsViewer
 
       initializer 'pdfjs_viewer-rails.load_static_assets' do |app|
-        app.middleware.unshift ::ActionDispatch::Static, "#{root}/public"
+        app.middleware.insert_before(::ActionDispatch::Static, ::ActionDispatch::Static, "#{root}/public")
```

Comment:
we are moving away from rails 3, so if no one else needs it we can just delete the pull reg.